src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -49,7 +49,7 @@ public AddressBook toModelType() throws IllegalValueException {
         AddressBook addressBook = new AddressBook();
         for (JsonAdaptedPerson jsonAdaptedPerson : persons) {
             Person person = jsonAdaptedPerson.toModelType();
-            if (addressBook.hasPerson(person)) {
+            if (addressBook.hasName(person)) {
```

Comment:
wait is this what you intend? isn't name not supposed to be unique

Reply:
Yup, names will be unique

Reply:
Nope, I just renamed it to hasName. I think hasPerson is probably better, what this does is it checks uniquepersonlist for a matching person (via checks for phone num, email, username) to find if theres a repeat